multisig/002_DeployFactoriesAndMulticall.sol

```diff
@@ -23,14 +22,10 @@ contract DeployFactoriesAndMulticall is MultisigProposal {
     address public usdcStrategyFactory;
     address public strategyMulticall;
 
-    constructor() {
+    constructor(address _addresses) {
```

Comment:
Let's not do that in the constructor and pass as a reference in the build/validate/deploy functions. Doing in the constructor we can't call the script using `forge script`